dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -403,14 +405,32 @@ await foreach (Completions completions in response.ConfigureAwait(false))
                 }
 
                 // Now, invoke the function, and add the resulting tool call message to the chat options.
+                FunctionResult functionResult = new(function) { Culture = kernel.Culture };
```

Comment:
It might be convenient to have an instance of Kernel in the filters.

Reply:
Thanks, added.